### DISCUSSION TITLE
My vehicle's  consumption is very high.

### DISCUSSION THREAD

**Discussion by zxc9019** [teslamate-org/teslamate#3179] [Q&A]
Why is my vehicle's idle energy consumption so high, at 0.38 miles per hour? It's about 3% per day, so if calculated this way, my car would be out of power in a month. The normal value is around 2 miles per day, so I'm far above the normal value. I called Tesla and they told me this is normal. Additionally, my vehicle should be in sleep mode, although there may be wake-ups, but the total wake time during the day is less than an hour. Are there any friends who have the same situation as me? Or is it because my vehicle's battery is lithium iron phosphate?

**Comment by ktsenvolt** [comment]
Sentry mode and climate protection off? Is no other third-party app other than TeslaMate?

**Reply by zxc9019** [reply]
Yes, I have turned off the sentry mode and climate protection, and even my car has only been purchased for two months.

**Reply by zxc9019** [reply]
And I changed the password as well, so besides Teslamate, no third-party programs have been involved.
